doc/application/application.rst

```diff
@@ -265,13 +265,19 @@ the :file:`zephyr` sub-directory of the build directory. (This is **not the
 Zephyr base directory**, which contains the Zephyr source code etc. and is
 described above.)
 
-* :file:`.config`, which contains the configuration settings
+* :file:`zephyr/.config`, which contains the configuration settings
```

Comment:
I think this was right originally, since the introductory paragraph says "...files will be written to
 the :file:`zephyr` sub-directory".

Reply:
It's deliberate. Thought it didn't hurt to repeat it.

Not hardcore about it though. Guess it can be misunderstood as `zephyr/zephyr/.config`, but not sure that'd happen.

Reply:
Changed my mind... probably not that helpful. Gone now.